EIPS/eip-4444.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 4444
```

Comment:
/me goes and checks to see if you farmed this EIP number...

Reply:
Crazy coincidence!

Reply:
🤨